pkgs/by-name/li/linuxhw-edid-fetcher/linuxhw-edid-fetch.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+set -eEuo pipefail
+test -z "${DEBUG:-}" || set -x
+# based on instructions provided in https://github.com/linuxhw/EDID/blob/98bc7d6e2c0eaad61346a8bf877b562fee16efc3/README.md
+
+usage() {
+  cat <<'EOF'
+Example invocations:
+  linuxhw-edid-fetcher /path/to/github.com/linuxhw/EDID PG278Q 2014 >edid.bin
+  edid-decode <edid.bin
+  parse-edid <edid.bin
+  cat edid.bin >/sys/kernel/debug/dri/0/DP-1/edid_override
+EOF
+}
+
+print() {
+  # shellcheck disable=SC2059
+  printf "${1}\n" "${@:2}"
+}
+
+log() {
+  print "${@}" >&2
+}
+
+find_displays() {
+  local script=(
+    "BEGIN { IGNORECASE=1 }"
+    "/${1}/"
+  )
+
+  for pattern in "${@:2}"; do
+    script+=('&&' "/${pattern}/")
+  done
+  cat "${repo}"/{Analog,Digital}Display.md | awk "${script[*]}"
+}
+
+to_edid() {
+  if ! test -e "$1"; then
+    log "$1 does not exist"
+    return 1
+  fi
+
+  # https://github.com/linuxhw/EDID/blob/228fea5d89782402dd7f84a459df7f5248573b10/README.md#L42-L42
+  grep -E '^([a-f0-9]{32}|[a-f0-9 ]{47})$' <"$1" | tr -d '[:space:]' | xxd -r -p
+}
+
+extract_path() {
+  awk '{ gsub(/^.+]\(</, ""); gsub(/>).+/, ""); print }'
+}
+
+main() {
+  if [[ $# == 0 ]] ; then
+    usage
```

Comment:
Nit: since this is an error condition, the usage message should probably go to stderr:

```suggestion
    usage >&2
```